src/rgw/rgw_crypt.cc

```diff
@@ -35,8 +35,6 @@ using namespace rgw;
 /**
  * Encryption in CTR mode. offset is used as IV for each block.
  */
-#warning "TODO: move this code to auth/Crypto for others to reuse."
```

Comment:
@amitkumar50 Please read the PR description above for the PR details for the movement.

First of all, adding warnings like this is not a good practise and the TODO should be specified in a tracker. Not within the code itself.